CONTRIBUTING.md

```diff
@@ -1,24 +1,28 @@
 # Contributors Guide
 
-## Bug Reports
+## Reporting Issues
 
 Please [open an issue](https://issues.serokell.io/newIssue?project=AD)
-if you find a bug. Note: you need to login (e. g. using your GitHub
-account) first.
+if you find a bug or have a feature request.
+Note: you need to login (e. g. using your GitHub account) first.
+Before submitting a bug report or feature request, make sure it hasn't already been submitted.
 
 The more detailed your report is, the faster it can be resolved.
+If you report a bug, please provide steps to reproduce this bug and revision of code in which this bug reproduces.
+
+## Branching Model
+
+This project uses a variation of the [OneFlow](https://www.endoflineblog.com/oneflow-a-git-branching-model-and-workflow) branching model with two branches. Naming of long-lived branches is different:
+* `develop` branch from OneFlow is called `master` in this repository.
+* `master` branch from OneFlow is called `production` in this repository.
 
 ## Code
 
 If you would like to contribute code to fix a bug, add a new feature, or
-otherwise improve Ariadne, pull requests are most welcome.
-
-Please make sure your contributions adhere to our coding guidelines:
+otherwise improve our project, pull requests are most welcome.
 
-*  Code must adhere to the [Style Guide](docs/code-style.md).
-*  Code should be documented with [Haddock](https://www.haskell.org/haddock/doc/html/index.html).
-*  Please refer to [this guide](https://chris.beams.io/posts/git-commit/) to write a good
-   Git commit message.
+Our pull request template contains a [checklist](./pull_request_template.md#white_check_mark-checklist-for-your-pull-request) of acceptance criteria for your pull request.
```

Comment:
Shouldn't the link start with `.github/`?

Reply:
Fixed, thanks.